.github/workflows/deploy.yml

```diff
@@ -92,3 +92,30 @@ jobs:
           no-cache: true
           build-args: DISTBALL=dist/target/camunda-zeebe-*-SNAPSHOT.tar.gz
           target: app
+          
+  notify-if-failed:
+    runs-on: ubuntu-latest
+    needs: [tests, deploy-snapshots, deploy-docker-snapshot]
+    if: failure()
+    steps:
+      - id: slack-notify
+        name: Send slack notification about build failure
+        uses: slackapi/slack-github-action@v1.18.0
+        with:
+          # For posting a rich message using Block Kit
+          payload: |
+            {
+              "text": "GitHub Action build result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
```

Comment:
Ah yes, thanks Slack, the word markdown clearly needed to be abbreviated :laughing:

Reply:
maybe it's markdown's cooler cousin, merkderwn :shrug: